layers/TileLayer.js

```diff
@@ -20,6 +22,10 @@ adv_map.layers.TileLayer = ccui.Widget.extend({
 
     this._super();
     this.navigator_data = [];
+    this.extra_data = {
+      BonusLevel: false,
+      GiftingLevel: false
```

Comment:
Ideally these things shouldn't here and should pass the view from game and adventure map should render the view with milestones

Reply:
done